.github/workflows/codeql-analysis.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
     - name: Autobuild
-      uses: github/codeql-action/autobuild@v1
+      uses: github/codeql-action/autobuild@v2
```

Comment:
Version bumping in Actions should probably be a separate PR.

Reply:
That's fair. Good catch -- I meant to at least put it in it's own commit. I'll go ahead and move to a separate PR for posterity

Reply:
PR: https://github.com/vmware-tanzu/function-buildpacks-for-knative/pull/138